File: extract.js
const fs = require('fs');
const readline = require('readline');

async function processLineByLine() {
  const fileStream = fs.createReadStream('C:\\Users\\yoruk\\.gemini\\antigravity\\brain\\ca31125c-e38f-45c4-a69a-24dbb2c0d0bc\\.system_generated\\logs\\transcript_full.jsonl');
  const rl = readline.createInterface({ input: fileStream, crlfDelay: Infinity });

  for await (const line of rl) {
    if (line.includes('TEST BORCU KAPATMA')) {
      const obj = JSON.parse(line);
      if (obj.type === 'USER_INPUT') {
        fs.writeFileSync('muhur_talimati.txt', obj.content);
        console.log('Yazıldı.');
        break;
      }
    }
  }
}
processLineByLine();

File: extract.py
import json
with open(r'C:\Users\yoruk\.gemini\antigravity\brain\ca31125c-e38f-45c4-a69a-24dbb2c0d0bc\.system_generated\logs\transcript_full.jsonl', 'r', encoding='utf-8') as f:
    for line in f:
        if 'TEST BORCU KAPATMA' in line:
            obj = json.loads(line)
            if obj.get('type') == 'USER_INPUT':
                with open('muhur_talimati.txt', 'w', encoding='utf-8') as out:
                    out.write(obj.get('content'))
                print('Yazıldı.')
                break
